build-tools/packages/build-tools/src/bumpVersion/utils.ts

```diff
@@ -57,3 +47,38 @@ export function prereleaseSatisfies(packageVersion: string, range: string) {
 	// Pretend that the current package is latest prerelease (zzz) and see if the version still satisfies.
 	return semver.satisfies(`${packageVersion}-zzz`, range);
 }
+
+/**
+ * Represents the different types of release groups supported by the build tools. Each of these groups should be defined
+ * in the fluid-build section of the root package.json.
+ */
+export enum MonoRepoKind {
```

Comment:
I think you can go ahead and mark this deprecated. Same for the related functions/type guards.

Reply:
done